web/src/components/Games.jsx

```diff
@@ -170,9 +171,9 @@ export default function Games({ loading, games, saveGame, saveQuestion, saveNewG
           </Select>
           <NewGameDialogue open={newGameOpen} onClose={() => setNewGameOpen(false)} submit={handleNewGame} />
         </Box>
-        {renderGames()}
+        {renderGames(loading)}
       </Grid>
-      {match && (
```

Comment:
Only render the Game form when a game exists at the index set in the URL. This was throwing an error before because the displayed game is chosen by its index in the game array, and that array is now affected (filtered) by the search bar.